test/functional/time_tracker_controller_test.rb

```diff
@@ -117,7 +117,7 @@ def setup
     } }, xhr: true
     assert_equal 2, TimerSession.count
     assert_equal TimerSession.last.finished, true
-    assert_in_delta TimerSession.last.timer_start, Time.zone.now - 1.hour, 1.second
+    assert_in_delta TimerSession.last.timer_start, Time.zone.now - 1.hour, 30.second
```

Comment:
```suggestion
    assert_in_delta TimerSession.last.timer_start, Time.zone.now - 1.hour, 30.seconds
```

Freezing time would make sense in these tests. They are opaque to me.

Reply:
Of course, plural is the only solution that makes sense in this condition.
Hmm, time freezing would make sense..